password-generator-manager/src/TableComponent.jsx

```diff
@@ -16,6 +16,15 @@ const TableComponent = () => {
     }
   };
 
+  function getFormattedDate(inputString) {
+    const date = new Date(inputString);
+    const year = date.getFullYear();
+    const month = String(date.getMonth() + 1).padStart(2, '0');
+    const day = String(date.getDate()).padStart(2, '0');
+    const formattedDate = `${year}-${month}-${day}`;
+    return formattedDate;
+  };
+
```

Comment:
I think this will complicate things. 
My suggestion would be to look at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat

It will take the input as single parameter and the result would be in the format expected. (**Date Time TimeZone**). And you can directly use that one liner directly inside <td>

Also, thank you for the PR, I'll create an issue for it and add your PR to it.

Reply:
Thank for replying! I'll check out the DateTimeFormat object!

Reply:
Hello, I updated the code to use the Intl.DateTimeFormat method, and made it a one-liner this is directly used in td tag. This is what it looks like for me now: 
![image](https://github.com/utk09/mlh-ghw-2023/assets/128189465/77536b05-9a91-486a-8e37-184506679cb8)
(I did a force push, so the old commit is no longer showing in the pull request history.)